docs/_layouts/examples.html

```diff
@@ -28,6 +33,7 @@
       }
     </style>
 
+    {% unless page.individual %}
```

Comment:
Styling for `u-baseline-grid` utility and the baseline grid toggle (positioning, switch, etc) are included in full vanilla but not in individual pattern CSS, so I don't render the switch on individual pattern example page for now.

We can consider bringing it on this page once we have this utility styles encapsulated, so we can include just these styles here.

Reply:
what do you mean by encapsulated? wouldn't it work by just adding:

```
@import 'utilities_baseline-grid';
@include vf-u-baseline-grid;
```

I'm a bit concerned about the baseline grid being seen as a design whim with debatable value.
There are, for example, 2 open issues to remove it.

From a design perspective, it is the only way to notice when something is a couple of pixels off.  For example, the pagination buttons were  ~ 3px taller than other buttons, something that wasn't noticed for months because in the real world you would never put them on the same row as a button. We only caught it because the grid made it obvious.

I realise it's a tall order to ask developers to actively QA for this, but it is something designers should IMO. Otherwise small discrepancies can silently accumulate.

Reply:
@lyubomir-popov Thanks for the comment.

In this example we don't want to add baseline grid styles into accordion pattern CSS, because we want to check if accordion styles work and build on their own.

From your comment it seems that CSS for baseline grid should be quite easy to build separately, so I'll try to do that.

Reply:
I added the baseline toggle on individual example page as well.

It doesn't use `p-switch` styles (because it's a separate pattern that is not included), but apart from that is fully functional.